tests/wpunit/Tribe/Tickets/Attendees_Test.php

```diff
@@ -248,6 +254,55 @@ public function test_event_details_top( Closure $fixture ): void {
 		}
 	}
 
+	/**
+	 * @test
+	 */
+	public function test_generate_filtered_list() {
```

Comment:
We may want to rename the test to remove the `test`prefix.For example, `it_should_generate_a_filtered_list`.

I also suggest possibly adding in a dataprovider that way you can test the scenario of one (or many) ticket with `generate_filtered_list()` and then with multiple tickets using `generate_filtered_list('all')`

It could help clean the code up a bit. You could also keep the expected data in an array and test that it matches what `generate_filtered_list()` returns for consistency.

Reply:
Thank you @redscar . About the name, isn't the `test_` prefix the proper one to use before the actual name of the function being tested? 

Cause that's the function we are testing here
`Tribe__Tickets__Attendees::generate_filtered_list()`

Of course we can use a `it_should_whatever` naming approach but I find the actual function names prepended with `test_` a more consistent way.

<hr>

We don't want multiple scenarios, just one scenario is enough: `more than 1 events`. And we check the function with one event and with all of them in the same context.

Reply:
I understand, the scenario may play a part into it. Usually if I add `@test` to the docblock I try to stay away from the `test_` prefix.

---
That makes sense, I was thinking more in regards to making sure that passing `all` or passing an Event ID to the function returns the expected output for both scenarios since we don't want the scenario where passing the Event ID to return all attendees.

Reply:
Added your suggestions, thanks!